### ISSUE TITLE
grepable format not really grepable

### ISSUE THREAD

**Issue by t0rzz** [robertdavidgraham/masscan#252]
Hello,
when outputting a scan result into any format (grepable, xml, json, binary, list) the port scan result is not unified for IP address when compared to nmap.

An example is:
**masscan:
- Host: 1.2.3.4 ()  Ports: 80/open/tcp////
- Host: 1.2.3.4 ()  Ports: 443/open/tcp////**

nmap:
**Host: 1.2.3.4 ()  Ports: 80/open/tcp//whatever///, 443/open/tcp//whatever///**

I believe this should be corrected as the result file, this way, is doubled and not grepable. I can't awk an IP address with a regex this way, while nmap allows me to use:
cat result | awk '/80\/open.*443\/open/{print $2}'

Hopefully I've been understood :)

**Comment by ravkr** [comment]
it is very grepable now... at least I think it is ;)

**Comment by berney** [comment]
@t0rzz I agree with @ravkr I prefer the masscan one line per IP/Port style over the nmap one IP many ports per line.

You should be able to modify you awk or use sed to process the many lines and extract the information you want.

**Comment by avilum** [comment]
Worked when i downgraded to 1.0.5 (built from this branch/tag).
Must be something in the new version
